canonical-web-and-design/ubuntu.com
# Build message for public clouds

## Done

- Updated the public cloud message to add a link to the contact page
- Disabled the “If you have more than one kind, you can add the others later” text if Azure or AWS is chosen 
(https://github.com/canonical-web-and-design/web-squad/issues/3600#issuecomment-756653195)

## QA

- go to https://ubuntu-com-9031.demos.haus/advantage/subscribe
- Select AWS or Azure and see that the new text shows, the contact link is working and the following text is disabled.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3601

## Screenshots

![image](https://user-images.githubusercontent.com/11927929/104718247-dfce1500-572a-11eb-9a45-34eab17ec615.png)
